CMake Error at CMakeLists.txt:69 (find_package):
  Could not find a package configuration file provided by
  "VulkanUtilityLibraries" with any of the following names:

    VulkanUtilityLibrariesConfig.cmake
    vulkanutilitylibraries-config.cmake

  Add the installation prefix of "VulkanUtilityLibraries" to
  CMAKE_PREFIX_PATH or set "VulkanUtilityLibraries_DIR" to a directory
  containing one of the above files.  If "VulkanUtilityLibraries" provides a
  separate development package or SDK, be sure it has been installed.

--- CMakeLists.txt.orig	2023-09-06 14:39:11 UTC
+++ CMakeLists.txt
@@ -66,7 +66,10 @@ endif()
     endif()
 endif()
 
-find_package(VulkanUtilityLibraries CONFIG REQUIRED)
+find_package(VulkanUtilityLibraries CONFIG)
+if (NOT TARGET Vulkan::UtilityHeaders OR NOT TARGET Vulkan::LayerSettings)
+    add_subdirectory(Vulkan-Utility-Libraries EXCLUDE_FROM_ALL SYSTEM)
+endif()
 
 option(BUILD_WERROR "Treat compiler warnings as errors")
 if (BUILD_WERROR)
